python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -786,6 +792,82 @@ def maskSaturatedPixels(self, badAmpDict, ccdExposure, detector, detectorConfig,
                 badAmpDict[ampName] = True
                 maskView |= maskView.getPlaneBitMask("BAD")
 
+        # Edge bleed masking
+        if self.config.doITLEdgeBleedMask and detector.getPhysicalType() == 'ITL':
```

Comment:
Please change to double-quotes (matching the rest of the file).  Though I think this whole function should be moved to `isrFunctions`.  Also, it needs a test.